components/eam/bld/namelist_files/use_cases/20TR_eam_chemUCI-Linoz.xml

```diff
@@ -83,8 +83,8 @@
 
 <!-- rad_climate -->
 <rad_climate>
-  'A:Q:H2O', 'N:O2:O2', 'N:CO2:CO2',
-  'A:O3:O3', 'N:N2O:N2O', 'N:CH4:CH4',
+  'A:H2OLNZ:H2O', 'N:O2:O2', 'N:CO2:CO2',
+  'A:O3:O3', 'N:N2OLNZ:N2O', 'N:CH4LNZ:CH4',
```

Comment:
Good catch, @mahf708 .

```suggestion
  'A:O3:O3', 'A:N2OLNZ:N2O', 'A:CH4LNZ:CH4',
```

Reply:
In v2, we don't use the ozone from the CMIP6 file.

I assume linoz_data (o3_clim) refers to one of the variables in the Linoz input file. Along with other Linoz input varialbes (production and loss terms, T, etc.) the Linoz v2 scheme calculates ozone in the stratosphere and when O3 enters the troposphere it is handled as a passive tracer with an e-folding sink in the lower boundary. The v2 radiation uses this Linoz v2 O3.

Reply:
Sounds like you might want to try something similar to double-calling radiation for aerosols via name list variable `rad_diag_1`. I suggested this for gas tracers before, but it seems only working for aerosols.